### PR TITLE
feat: hero landing page

### PR DESCRIPTION
**Pull request by ankithans**
- https://ankithans.github.io/graphql-link/
- hero landing page
- announcement bar

#48

### CODE REVIEW

File: website/docusaurus.config.js [hunk 1]
```diff
@@ -6,17 +6,30 @@ module.exports = {
...
   plugins: ["docusaurus-tailwindcss-loader"],
+
   themeConfig: {
+    colorMode: {
+      disableSwitch: true,
+      defaultMode: "dark",
+    },
+    announcementBar: {
+      id: "support_us",
+      content: "This website is under construction 🚧",
```
Comment (wtrocki) [inline website/docusaurus.config.js:21]: I would not add this

File: website/docusaurus.config.js [hunk 2]
```diff
@@ -6,17 +6,30 @@ module.exports = {
   baseUrl: "/graphql-link/",
   onBrokenLinks: "ignore",
   onBrokenMarkdownLinks: "warn",
-  favicon: "img/logo.ico",
+  favicon: "img/logo_without_text-removebg-preview.png",
```
Comment (wtrocki) [inline website/docusaurus.config.js:9]: There is a lot of images added in PR and not every image is used. can we remove those?

File: website/docusaurus.config.js (hunk 1 above)
Comment (ankithans) [reply website/docusaurus.config.js]: removed

File: website/docusaurus.config.js (hunk 2 above)
Comment (ankithans) [reply website/docusaurus.config.js:9]: done

**Pull request merged**